### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

File: css/card.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
+.card-container {
+    border-radius: 5px;
+    background-color: #ffffffed;
+    width: 23rem;
+    height: 24.3rem;
+    box-shadow: 0px 2px 5px 1px palevioletred;
```
Comment (Naveenchandar) [inline css/card.css:6]: Please use hex code values for consistency.

File: css/card.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    font-weight: 700;
+}
+.card-header:hover {
+    background-color: var(--card-hover-light);
+    cursor: pointer;
+}
+.footer-btn:hover , .footer-icons:hover {
+    background-color: var(--card-hover-light);
+    cursor: pointer; 
+}
```
Comment (Naveenchandar) [inline css/card.css]: You can merge these to single style.

File: css/card.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    padding: 1.5rem;
+    cursor: pointer;
+    color: var(--primary-blue);
+}
+#icon-dismiss {
+    transition-timing-function: linear;
+}
+.has-dismiss {
+    z-index: -1;
+    position: absolute;
```
Comment (Naveenchandar) [inline css/card.css:103]: You can use 'position-abs' class mentioned in utils class

File: css/card.css (hunk 1 above)
Comment (anupkgurung) [reply css/card.css:6]: done !!

File: css/card.css (hunk 2 above)
Comment (anupkgurung) [reply css/card.css]: sure !!

File: css/card.css (hunk 3 above)
Comment (anupkgurung) [reply css/card.css:103]: yeah will move it all together !!

File: css/card.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+}
+.card-desc--overlay {
+    margin-top: 1.4rem;
+}
+.card-header--horizontal {
+    flex-direction: row;
+    gap: 5%;
+}
+.card-footer {
+    display: flex;
```
Comment (Naveenchandar) [inline css/card.css:46]: suggestion:
If flex is repeated more often in components, You can move flex properties to utility file.

Comment (anupkgurung) [reply css/card.css:46]: yeah it's there in utils. i'll have to refactor all in dev.

**Review by Naveenchandar — APPROVED** [review]
Looks good.

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

**Pull request merged**